### DISCUSSION TITLE
Why there is no wheel for Linux? Can it be created?

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [python-pillow/Pillow#6475] [Q&A] [answered]
Why the package is always built from source?

I can not waste time In CI/CD to build packages from source. Is it possible at all to build a wheel for Pillow so I can quickly install it in my Docker? Maybe the are some other way to "build" the package and distribute it?

**Comment by hugovk** [comment]
It's not always built from source; we released 57 wheel files for 9.2.0, 31 of which are manylinux and four are musllinux (and 11 each for Windows and macOS):

https://pypi.org/project/Pillow/9.2.0/#files

What Python version and OS are you using on your CI? Can you share logs of the installation/build?

**Reply by nulano** [reply]
Don't forget that installing the pre-built wheels requires installing the `wheel` package first, which is not always pre-installed in Docker images.

**Comment by simon-liebehenschel** [comment, marked as answer]
I am really sorry for my question! I confused "Pillow-SIMD" with the "Pillow". Yes, "Pillow" is installed without building, but "Pillow-SIMD" is built.
My question becomes off-topic. Thanks for your patience.

PS: I will try to build "Pillow-**SIMD**" by myself for further reusing in Docker in CI/CD. I have no experience with building, so I hope this is possible.
